ansible/roles/master/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 dependencies:
     - { role: common }
+    - { role: docker }
```

Comment:
No, I'm asking, since you already pull in the docker role via the opencontrol roles, do you also need to pull it in here? I don't think you can make this conditional really.  I'm asking do we need it here, or was it taken care of in other roles?